src/js/components/SchemaForm.js

```diff
@@ -53,6 +53,13 @@ class SchemaForm extends React.Component {
     this.setState({renderGemini: true});
   }
 
+  componentWillUnmount() {
+    // Unschedule all validation if component unmounts.
+    if (this.timer) {
+      clearTimeout(this.timer);
```

Comment:
This is intentional since we will need the id for the `cancelTimeout` if the component unmounts before the timeout has been triggered.